### ISSUE TITLE
Version 1.1.16: Reference could not be resolved using parser.parseRAMLSync including json type

### ISSUE THREAD

**Issue by glodieu** [raml-org/raml-js-parser-2#671]
This working fine using parser version 1.1.15.
Received this error
"message": "Invalid JSON schema: Reference could not be resolved: /tmp/Complex.json#Foo /tmp/Complex.json#Foo"
using parser method parseRAMLSync like this 
```
const myFsResolver = {
  content: function (filePath) {
    return fs.readFileSync(filePath, 'UTF8');
  },
  contentAsync: function (filePath) {
    return new Promise(function (resolve, reject) {
      try {
        resolve(fs.readFileSync(filePath, 'UTF8'));      
      }
      catch (e) {
        reject(e);
      }
    });
  }
};
const options = {"attributeDefaults":false,"rejectOnErrors":true,"validate":true, "fsResolver": myFsResolver};
const parsedData = parser.parseRAMLSync(data, options);
```

with RAML data 
```
#%RAML 1.0
title: Swagger Petstore
version: 1.0.0
baseUri: 'http://petstore.swagger.io/v2'
protocols:
  - HTTP
description: This is a sample server Petstore server.
/pet:
  displayName: pet
  post:
    displayName: addPet
    description: Add a new pet to the store
    responses:
      '405':
        description: Invalid input
types:
  Foo:
    type: !include ../../types/Complex.json#Foo
```

and  /tmp/Complex.json
```
{
  "Foo": {
    "type": "object",
    "description": "Foo details",
    "properties": {
      "id": { "type": "integer" },
      "name": { "type": "string" },
      "ownerName": { "type": "string" }
    },
    "required": [ "id", "name" ]
  },
  "Person": {
    "type": "object",
    "description": "Person details",
    "properties": {
      "firstName": { "type": "string" },
      "lastName": { "type": "string" },
      "nationality": { "type": "string" }
    },
    "required": [ "firstName", "lastName" ]
  }
}
```

**Comment by KonstantinSviridov** [comment]
Hi, @glodieu 

First, the "Complex.json" has wrong location. According to the reference `!include ../../types/Complex.json#Foo`, it should be located in `/types`, not in `/tmp`.

Second, right now the parser can not process the reference `type: !include ../../types/Complex.json#Foo` without `/` after `#`. So, for now, the reference should be `type: !include ../../types/Complex.json#/Foo`. This will be fixed in 1.1.17.

Regards, Konstantin.

**Comment by glodieu** [comment]
Hi @KonstantinSviridov, 
First part: yes, you're right regarding path between /types and /tmp it's my fault.
Second part: thanks!
